[quant][pyper] Rename the sparse argument for embedding_bag ops

supriyar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#46003 [quant][pyper] Rename the sparse argument for embedding_bag ops**
* #45997 [quant][pyper] Set sparse to False for embedding_bag ops in graph mode

Summary:
sparse is confusing because itt is used in training for sparse gradients

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24178248](https://our.internmc.facebook.com/intern/diff/D24178248)

supriyar committed:
[quant][pyper] Rename the sparse argument for embedding_bag ops
Summary:
sparse is confusing because itt is used in training for sparse gradients

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][pyper] Rename the sparse argument for embedding_bag ops"


Summary:
sparse is confusing because itt is used in training for sparse gradients

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24178248](https://our.internmc.facebook.com/intern/diff/D24178248)

[ghstack-poisoned]

supriyar committed:
Update on "[quant][pyper] Rename the sparse argument for embedding_bag ops"


Summary:
sparse is confusing because itt is used in training for sparse gradients

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24178248](https://our.internmc.facebook.com/intern/diff/D24178248)

[ghstack-poisoned]

supriyar committed:
Update on "[quant][pyper] Rename the sparse argument for embedding_bag ops"


Summary:
sparse is confusing because itt is used in training for sparse gradients

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24178248](https://our.internmc.facebook.com/intern/diff/D24178248)

[ghstack-poisoned]

vkuzo reviewed (approved):
lg, I'm assuming there are no use cases yet that have to be migrated?

supriyar committed:
Update on "[quant][pyper] Rename the sparse argument for embedding_bag ops"


Summary:
sparse is confusing because itt is used in training for sparse gradients

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24178248](https://our.internmc.facebook.com/intern/diff/D24178248)

[ghstack-poisoned]

supriyar commented:
> lg, I'm assuming there are no use cases yet that have to be migrated?

Not in OSS, there are some internal use cases but they aren't setting this field just yet.

supriyar committed:
Update on "[quant][pyper] Rename the sparse argument for embedding_bag ops"


Summary:
sparse is confusing because itt is used in training for sparse gradients

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24178248](https://our.internmc.facebook.com/intern/diff/D24178248)

[ghstack-poisoned]
